core/cppgc.rs

```diff
@@ -43,10 +43,17 @@ pub fn make_cppgc_object<'a, T: GcResource + 'static>(
   obj
 }
 
+// TODO(littledivy): https://github.com/denoland/rusty_v8/pull/1505
+#[repr(C)]
+struct InnerMember {
+  inner: [usize; 2],
+  ptr: *mut (),
+}
+
 #[allow(clippy::needless_lifetimes)]
 pub fn try_unwrap_cppgc_object<'sc, T: GcResource + 'static>(
   val: v8::Local<'sc, v8::Value>,
-) -> Option<&'sc T> {
```

Comment:
This is not safe anymore. Please now mark as unsafe. The caller has to guarantee that there are not more than one user currently called `try_unwrap_cppgc_object` on this object.

Reply:
I'll add an unsafe variant that returns `&mut`